src/form.tsx

```diff
@@ -47,7 +47,8 @@ const Form: React.FC<FormProps> = ({
     const fieldRegistrations = React.useRef(new Map<string, FieldRegistration>());
     const formRef = React.useRef<HTMLFormElement | null>(null);
     const {texts, t} = useTheme();
-    const id = useAriaId(idProp);
+    const reactId = React.useId();
+    const id = idProp ?? reactId;
```

Comment:
I would use `||` here and consider an empty id string as undefined

Reply:
done